playbooks/provisioner/virsh/main.yaml

```diff
@@ -294,6 +294,14 @@
             ansible_ssh_host="{{ item }}"
         with_items: groups['openstack_nodes']
 
+      - name: update the ssh host name of Loadbalancer machine
+        add_host:
+           name="{{ item }}"
+           ansible_ssh_host="{{ item }}"
+        with_items: groups['loadbalancer']
+        ignore_errors: True
```

Comment:
I think it would be better here to use condition and call "add_host" only when loadbalancer is defined